src/dev/src/app/combobox/optional-menu.html

```diff
@@ -9,7 +9,7 @@
   Select 2.0 implementation is still incomplete. DO NOT USE.
   All issues related to clr-select will be closed until the implementation is completed.
 -->
-<h6>1. Menu provided by the consumer</h6>
```

Comment:
Reason?

Reply:
AOT errors since combobox is not actually a public in clr-angular